### ISSUE TITLE
Tamil font can supoort subscripts 2, 3 and 4

### ISSUE THREAD

**Issue by sridatta1** [TiroTypeworks/Indigo#52]
Currently the superscripts are supported, subscripts can also be supported as they are also used in some books.
![image](https://user-images.githubusercontent.com/23263723/207642728-0073e17c-57ed-410b-8f3b-767681900813.png)

**Comment by tiroj** [comment]
The superscripts were included because they are used in the Murty volumes. If I were to add support for the subscripts, I would want to do it for all the Tiro Indic collection fonts, to retain a common model. Will think about this, but perhaps not make it a priority.

**Comment by sridatta1** [comment]
Yes, not a priority feature request, can be taken much later if necessary.
I raised this issue for only Tamil, as they are used for transliteration of Sanskrit ( the 2nd, 3rd and 4th consonants of the vargas which Tamil lacks).
